apps/frontend/src/lib/chip/leaderboard.ts

```diff
@@ -19,7 +64,7 @@ export async function updateLeaderboardEntry(
 
   try {
     const response = await fetch(
-      `${BASE_API_URL}/user/update_leaderboard_entry`,
+      `${BASE_API_URL}/chip/update_leaderboard_entry`,
```

Comment:
NOTE: I changed the leaderboard routes from user to chip. All the frontend and backend leaderboard code already lives in `chip/` directories, not `user/`. This feels sensible to me, I think leaderboards should be "owned" by the chip issuer.

Reply:
yep this was a great choice, appreciate the thought there